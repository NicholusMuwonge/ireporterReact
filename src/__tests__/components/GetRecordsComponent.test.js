import React from 'react';
import { shallow } from 'enzyme';
import RecordFetchComponent from '../../components/records/Recordfetch';


describe('<RecordFetchComponent />', () => {
  const props = {
    record: {
      body: 'sleek',
    },
  };

  it('it should render component without fail', () => {
    const wrapper = shallow(<RecordFetchComponent {...props} />);
    expect(wrapper).toMatchSnapshot();
  });
});
