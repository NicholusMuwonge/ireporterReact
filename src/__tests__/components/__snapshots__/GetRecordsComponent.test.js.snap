// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<RecordFetchComponent /> it should render component without fail 1`] = `ShallowWrapper {}`;
